website/_kb/user-guide/alerts/setting-up-an-alert.md

```diff
@@ -20,37 +20,34 @@ In the Alerts view, you'll see all existing alerts.
+5. Leave `Rearm Seconds` empty to get 1 alert when the status changes from Triggered to OK. Or enter a number to be alerted every time the query runs.
 6. Save.
 7. Define alert destinations - email, [Slack]({% link _kb/user-guide/alerts/slack-alert-destination.md %}), Mattermost, HipChat and webhooks are supported.
 
 <img src="/assets/images/docs/gitbook/alert_destination.png" width="60%">
 
 ## Alert Status & Frequency
 
-The Alert Status is checked every time the query is executed - (alerts only work with scheduled queries).
+The Alert Status is checked every time the query is executed, but an Alert notification is sent only for scheduled queries.
 
 Alerts have 3 Status Types:
 
-* `TRIGGERED` - the value you set the alert for is triggered (if you set your alert to trigger when the value of "cats" is greater than 1500 as long as it's above 1500 your alert is triggered)
-* `OK` - the value you have set to trigger the alert is not reached for now (might happen after the alert was triggered or before it was ever triggered, if your "cats" value is now 1470 your alert will show as OK)
-* `UNKNOWN` - you should see this status once you have set your alert and it wasn't yet checked. To make your alert in the know, run the query it is linked to after setting the alert.
+* `TRIGGERED` - Your alert has been tripped because the current value of the desired field matches your criteria. For example, if you set your alert to trigger when the value of "cats" is greater than 1500, your alert will remain triggered as long as the query shows more than 1500 "cats".
+* `OK` - Your alert is not tripped as of the most recent query execution. This can happen after the alert has been triggered (in which case the status will change from TRIGGERED to OK) or before it was ever triggered. If your "cats" value is now 1470 your alert will show as OK.
+* `UNKNOWN` - You should see this status when you first set up your alert because it hasn't been checked yet. You can run the query to confirm that the Alert logic works. Manually running the query can change the Alert status but will not trigger a notification.
 
-### Get an Alert Upon Status Change
+#### Get an Alert Upon Status Change
 
-If you leave the REARM value empty alerts will only be sent when the status
-changes (from OK to Triggered or vice versa).
+If you leave the REARM value empty alerts will only be sent when the status changes (from OK to Triggered or vice versa).
 
 #### Get an Alert Everytime the Query Runs
 
-To get an alert every time the query runs, set the REARM value - the value is
-for seconds that pass since the system detects a change until it sends the
-alert, a 1-minute "delay" would require entering '60' in the REARM field.
+To get an alert every time the query runs, set the REARM value - the value is for seconds that pass since the system detects a change until it sends the alert, a 1-minute "delay" would require entering '60' in the REARM field.
```

Comment:
The value isn't a delay, in the sense of how long the system waits until it sends the message. It's the time it waits since the previous time it sent a notification until it sends a new one.

An example:

* You have an alert based on a query that runs every 5 minutes.
* If REARM is empty, then when the alert changes state to TRIGGERED you will receive a single notification. The next time the system will notify you will be when the alert switches state to OK or UNKNOWN.
* If you want to receive follow up notification when the alert is still TRIGGERED without a state change, you can set the REARM value. In this example, if you set it to 60 seconds, you will receive a notification every time the query runs (5 minutes). But if you want more time between notification, let's say you want one every hour, you can set it to 3600 seconds and then the next notification (after the first one) will be an hour later, assuming the alert is still triggered.